Holovibes/includes/compute/registration.hh

```diff
@@ -0,0 +1,239 @@
+/*! \file
+ *  \brief Implementation of algorithms used for registration.
+ *  The registration performs computations as follows:
+ *  - Apply a circular mask to each images where each values outside of the circle is 0 and each
+ *  values inside the circle is 1. The radius is to be chosen by the user. (hardcoded for now)
```

Comment:
Don't forget to remove the hardcoded fro now when the settings will be supported